plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2954,6 +2954,17 @@ public void testSplitPruningForFilterOnNonPartitionColumn(DataMappingTestSetup t
         }
     }
 
+    @Test
+    public void testGetTableProperties()
+    {
+        assertUpdate("CREATE TABLE test_iceberg_get_table_props (x BIGINT) WITH (format='PARQUET')");
```

Comment:
Please drop `WITH (format='PARQUET')`.
If you have a look at https://github.com/trinodb/trino/blob/master/plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java#L138 you notice that the file format is specified at the connector level to be either `ORC` or `PARQUET` depending on the concrete test class in which the test is executed.